App structure new

kpatel1209 opened the pull request:
- restructured the screen navigation flow and added a drawer where the user can click on the hamburger menu when logged in to display the menu or even swipe right from the left edge of their device.  will need to test out functionality with new screens when added and create proper routing.

- implemented and tested a ReportOrView screen that appears after submitting an issue where we are now able to select Report Another Issue which takes us back to the Report Issues screen, or select View Issues which takes us to the View Issues screen

kpatel1209 committed:
screen navigation is paritally working

kpatel1209 committed:
submit button on create account page functional

kpatel1209 committed:
navigate screen work in progress

kpatel1209 committed:
trying to implement scroll view on create account page

kpatel1209 committed:
created test branch for screen nav testing

kpatel1209 committed:
merging branches

kpatel1209 committed:
testing out new route structure

kpatel1209 committed:
new screen nav is a work in progress

kpatel1209 committed:
trying to figure out how to update tab and drawer names so it displays more than 1 word

kpatel1209 committed:
tab and drawer names in WIP

kpatel1209 committed:
cleaned up code on viewissues file

kpatel1209 committed:
working on tab names

kpatel1209 committed:
fixed title naming convention on drawer

kpatel1209 committed:
fixed title naming convention on bottom tab navigator

kpatel1209 committed:
cleaning up code

kpatel1209 committed:
used prettier to format code

kpatel1209 committed:
prettied up code again

kpatel1209 committed:
code clean up

kpatel1209 committed:
adjusted color of tab bar icons and labels

heincm reviewed (dismissed):
Looks good. I'm assuming you're keeping the commented out files as a "just in case" while we figure out navigation. Will probably want to look into deleting those files altogether on the next go round.

kpatel1209 committed:
updated icons for bottom nav bar

kpatel1209 committed:
switching to ReportOrView screen is work in progress

kpatel1209 committed:
testing out sampel reportorview screen

kpatel1209 committed:
created a Dashboard navigator that houses the View and Report screens to fix recent screen nav issues

kpatel1209 committed:
able to navigate to ReportOrView screen after submtting issue

kpatel1209 committed:
able to choose report another issue or view issues on the ReportOrView screen

kpatel1209 committed:
removed navigation folder and associated files, cleaned up code and removed commented out code

kpatel1209 commented:
I removed that navigation folder and it's files since we aren't going to be using them and also removed my code that was commented out since I no longer need them.  Screen navigation has been tested and appears to be working as we expected.

kpatel1209 committed:
added initial route to reportstack nav
